src/scripts/newAdapterCommand.ts

```diff
@@ -186,49 +186,6 @@ export function newAdapterCommand(
             default: template ? template : undefined,
             filter: (input: string) => Templates[input],
           },
-          {
-            type: 'input',
-            name: 'adapterClassName',
-            message:
-              'What PascalCase name should be used for the adapter class?',
-            default: ({
-              protocolKey,
-              productId,
-            }: {
-              protocolKey: string
-              productId: string
-            }) =>
-              `${inputProtocolKey ?? protocolKey}${pascalCase(
-                productId,
-              )}Adapter`,
-            validate: async (
-              input: string,
-              {
-                protocolId,
-                productId,
-              }: { protocolId: string; productId: string },
-            ) => {
-              if (!isPascalCase(input)) {
-                return 'Value must be PascalCase'
-              }
-
-              if (!input.endsWith('Adapter')) {
-                return 'The adapter class should end up with the word Adapter'
-              }
-
-              if (
-                await adapterFileExists(
-                  inputProtocolId ?? protocolId,
-                  productId,
-                  input,
-                )
-              ) {
-                return 'An adapter with that name already exists'
-              }
-
-              return true
-            },
-          },
```

Comment:
Removed question and setting adapter class and file name based on `protocolId` and `productId`.

We are already checking on `productId` for uniqueness.